SPECS/SymCrypt-OpenSSL/SymCrypt-OpenSSL.spec

```diff
@@ -41,6 +41,9 @@ cmake   .. \
 
 cmake --build .
 
+%check
+./bin/SslPlay/SslPlay
```

Comment:
Nit: `%check` section should go after %install, since that's the order they're ran in

Reply:
Ah - I was following the pattern from OpenSSL's SPEC - fixed